app/konnect/dev-portal/audit-logging/webhook.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Set up an audit log webhook for Dev Portal
+content_type: how-to
+---
+
+You can use the {{site.konnect_short_name}} UI or the [Audit Logs](/konnect/api/audit-logs/latest/) and [Portal Management](/konnect/api/portal-management/latest/) APIs to configure webhooks for [audit logging](/konnect/dev-portal/audit-logging/). 
+
+{% include_cached /md/konnect/audit-logging/webhook-overview-prereq-siem-config.md desc='Dev Portal' %}
+
+
+## Create a webhook
+
+{% navtabs %}
+{% navtab Konnect UI %}
+Before you configure the webhook, you must first create an audit log destination. This allows you to set your audit log destination (the endpoint URL for your SIEM provider) and reuse it. 
+
+1. From {% konnect_icon organizations %} [**Organization**](https://cloud.konghq.com/organization) in the sidebar, click **Audit Logs Setup**.
+1. On the **Webhook Destination** tab, click **New Webhook** and configure the following:
+   * **Name**: The name you want to display for the audit log destination.
+   * **Endpoint**: The external endpoint that will receive audit log messages. 
+   * **Authorization Header**: The authorization type and credential to pass to your log collection endpoint. 
+    {{site.konnect_short_name}} will send this string in the `Authorization` header of requests to that endpoint.
+
+     For example, if you are setting up the webhook for Splunk, you could provide a Splunk access token: 
+     `"authorization":"Splunk example-token12234352535235"`.
+        
+    * **Log Format**: The output format of each log message. Can be CEF or JSON.
+    * **Disable SSL Verification**: Disables SSL verification of the host endpoint when delivering payloads. We recommend disabling SSL verification only when using self-signed SSL certificates in a non-production environment as this can subject you to man-in-the-middle and other attacks.
+1. To configure the Dev Portal audit log webhook, navigate to {% konnect_icon dev-portal %} [**Dev Portal**](https://cloud.konghq.com/portal) in the sidebar.
```

Comment:
Users can also navigate to the "Dev Portal" sub tab in "Organizations -> Audit Logs Setup" and associate a destination with a dev portal.

Reply:
good call out! I'll make sure to add a note to each UI instruction set that explains where to go in the Org settings to configure the Dev Portal audit logs as an alternative method.